src/plots/gl3d/scene.js

```diff
@@ -448,7 +454,9 @@ proto.render = function() {
             pointData.bbox = bbox[0];
         }
 
-        if(selection.buttons && selection.distance < 5) {
+        if(selection.buttons && selection.distance < 5 && !tabletmode) {
+            gd.emit('plotly_click', eventData);
+        } else if(tabletmode && selection.distance < 5) {
```

Comment:
Thanks for the PR. 
Could we possibly combine the two if statements for `plotly_click` like this?
```js
        if(selection.distance < 5 && (selection.buttons || tabletmode)) {
            gd.emit('plotly_click', eventData);
        } else {
            gd.emit('plotly_hover', eventData);
        }
```

Reply:
Thank you! I just took your review which only changed draftlogs but the test passed. The same thing happened between [82604eb](https://github.com/plotly/plotly.js/pull/6563/commits/82604eb29d7b9347d0d6ae65880617bdc6b7c09e) and [735f8f2](https://github.com/plotly/plotly.js/pull/6563/commits/735f8f27cd7a4808588d517c7158a17e47abd304)  

In that commit I just eslinted but the test passed too.  [webgl-jasmine](https://circleci.com/gh/plotly/plotly.js/188200?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary) fails sometimes. weird...